src/ui/public/styles/variables/depth.less

```diff
@@ -0,0 +1,5 @@
+/**
+ * 1. The local nav contains tooltips which should pop over the filter bar.
+ */
+@globalDepthFilterBar: 2; /* 1 */
+@globalDepthLocalNav: 3; /* 1 */
```

Comment:
Ahhh, shoot you're right. I was thinking they had fixed positioning for some reason.

I think we still need to group these variables together because they make more sense when viewed in the same context. But the name definitely doesn't make sense.

I think the long-term solution is to:

1. Move the `filter_bar` directory into the `kbn_top_nav` directory
2. Import the `filter_bar` into the `kbn_top_nav` instead of in `autoload/modules.js`.
3. Create a `kbn_top_nav/kbn_top_nav.less` file which contains these variables and the relevant `kbn_top_nav` styles from `base.less`.
4.Rename these variables to just be `filterBarDepth` and `localNavDepth`.

It's a little bit more work, but it will be an improvement to our module organization.